standardfiles/ide/.github/lock.yml

```diff
@@ -0,0 +1,8 @@
+---
+daysUntilLock: 30
```

Comment:
Do we really want to roll out locking. It causes enough e-mail spam that it tends to result in folks unfollowing repos which seems to be the last thing we want.

Reply:
I can remove it if we want to? 

I think most are on 30 days now anyway, but your call?